Common/cpp/Registries/EventHandlerRegistry.cpp

```diff
@@ -6,17 +6,20 @@ namespace reanimated {
 void EventHandlerRegistry::registerEventHandler(
     std::shared_ptr<WorkletEventHandler> eventHandler) {
   const std::lock_guard<std::mutex> lock(instanceMutex);
-  eventMappings[eventHandler->eventName][eventHandler->id] = eventHandler;
-  eventHandlers[eventHandler->id] = eventHandler;
+  const std::string &eventName = eventHandler->getEventName();
+  auto handlerId = eventHandler->getHandlerId();
+  eventMappings[eventName][handlerId] = eventHandler;
```

Comment:
Does it make sense to use `std::make_pair(eventName, handlerId)` as key here to avoid double lookup?